uaclient/config.py

```diff
@@ -68,6 +68,7 @@ class UAConfig:
         "status-cache": DataPath("status.json", False),
         "notices": DataPath("notices.json", False),
         "marker-reboot-cmds": DataPath("marker-reboot-cmds-required", False),
+        "services-once-enabled": DataPath("services-once-enabled", False),
```

Comment:
This almost feels like a simple binary auditing of a service status. It's may be a bit too early for us to invest a lot of time in fleshing that out for this one case. But, we will likely need to address this if we start getting to generic auditing or config changes. I definitely don't think we want N files in /var/lib/ubuntu-advantage/private for every attrubute we are tracking, but, since this is the first case. Let's keep it simple.